.bin/srclib-javascript

```diff
@@ -181,9 +181,7 @@ parser.command("depresolve")
   })
   .help("resolves dependencies of a CommonJS package");
 
-var hardcodedRepoURLs = {
```

Comment:
But we don't know it's node.  An NPM package doesn't necessarily need node to run (it could be a frontend package, or not even javascript!)

And we don't know if it's node, iojs, etc...